File: src/main/java/uk/gov/justice/digital/hmpps/riskprofiler/datasourcemodel/Pras.java
package uk.gov.justice.digital.hmpps.riskprofiler.datasourcemodel;

import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

@Data
@Builder
@EqualsAndHashCode(of = { "nomisId"})
@ToString
public class Pras implements RiskDataSet {
    public static int NOMIS_ID_POSITION = 4;
    private String nomisId;

    public String getKey() {
        return nomisId;
    }

}

File: src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/datasourcemodel/Pras.kt
package uk.gov.justice.digital.hmpps.riskprofiler.datasourcemodel

data class Pras(private val nomisId: String) : RiskDataSet {

  override fun getKey() = nomisId

  companion object {
    @JvmField
    var NOMIS_ID_POSITION = 4
  }
}
